jsonify_stored/models/jsonify_stored_mixin.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import json
+from itertools import groupby
+
+from odoo import api, fields, models
+from odoo.tools.misc import split_every
+
+
+class JsonifyStoredMixin(models.AbstractModel):
+    """This mixin aims to provide the ability to store jsonified data on model."""
+
+    _name = "jsonify.stored.mixin"
+
+    def _ir_export_id_domain(self):
+        return [("resource", "=", self._name)]
+
+    # TODO writeable, compute default value, fitler by current model
+    ir_export_id = fields.Many2one(
```

Comment:
For now I would not make this field stored. You can make it computed and provide a `_jsonify_get_exporter` hook.
In the case of c_search_engine, it will return `self.index_id.exporter_id`